app/config/AppConfig.scala

```diff
@@ -29,9 +29,10 @@ class AppConfig @Inject()(
   @Named("appName") serviceIdentifier: String
 ) {
 
-  runModeConfiguration
-    .getOptional[String]("mongodb.uri")
-    .map(uri => MongockConfig(uri))
+//  // TODO - fix or remove
```

Comment:
Should we try to update the latest `java-mongo-driver`?

Reply:
We can try all sorts of things.  But after we get the pipeline unblocked.   There's stuff already deployed to QA that is wrong.